app/views/cases/show.html.erb

```diff
@@ -33,31 +33,38 @@
-      <br/><small><%= @this_case.date.to_s(:long) %></small>
-          <% if @this_case.agencies.present? %>
-            <p>Agencies:
-              <% @this_case.agencies.each_with_index do |agency, index| %>
-                <%= link_to agency.name, agency_path(agency) %>
-                <% unless index == @this_case.agencies.size - 1 %>
-                  <%= "|" %>
+      <span class="page-summary">
+        <h1 class="page-header case-title"><%= @subjects.first.name %>, <%= @subjects.first.age ? "#{@subjects.first.age}" : "" %>
+        <br>
+            <small><%= @this_case.city %>, <%= @this_case.state.name %></small>
+        <br/><small><%= @this_case.date.to_s(:long) %></small>
+            <% if @this_case.agencies.present? %>
+              <p>Agencies:
+                <% @this_case.agencies.each_with_index do |agency, index| %>
+                  <%= link_to agency.name, agency_path(agency) %>
+                  <% unless index == @this_case.agencies.size - 1 %>
+                    <%= "|" %>
+                  <% end %>
+                <% end %>
+              </p>
+            <% end %>
+            <div>
+              <% if current_user %>
+                <% unless current_user.following?(@this_case) %>
+                  <%= link_to 'Follow This Case', follows_case_path(@this_case), method: :post, class: "btn-lg btn-primary text-center", data: { toggle: 'popover', container: 'body', trigger: 'hover', placement: 'bottom', content: 'Click here to receive an email whenever this case is updated.' } %>
                 <% end %>
-              <% end %>
-            </p>
-          <% end %>
-          <div>
-            <% if current_user %>
-              <% if current_user.following?(@this_case) %>
               <% else %>
-                <%= link_to 'Follow This Case', follows_case_path(@this_case), method: :post, class: "btn-lg btn-primary text-center", data: { toggle: 'popover', container: 'body', trigger: 'hover', placement: 'bottom', content: 'Click here to receive an email whenever this case is updated.' } %>
+                <%= link_to 'Follow This Case', follows_case_path(@this_case), method: :post, class: "btn-lg btn-primary text-center", data: { toggle: 'popover', container: 'body', trigger: 'hover', placement: 'bottom', title: 'Login to Follow this case!', content: 'Click here to receive an email whenever this case is updated.' } %>
               <% end %>
-            <% else %>
-              <%= link_to 'Follow This Case', follows_case_path(@this_case), method: :post, class: "btn-lg btn-primary text-center", data: { toggle: 'popover', container: 'body', trigger: 'hover', placement: 'bottom', title: 'Login to Follow this case!', content: 'Click here to receive an email whenever this case is updated.' } %>
-            <% end %>
-          </div>
-      </h1>
+            </div>
+            
+        </h1>
+        <% if current_user && current_user.admin? %>
```

Comment:
@danuluma Can you change this conditional to match the way we display the edit profile button on the user show page?

Reply:
Sure, let me do that

Reply:
We did add a case policy so that should make it a bit easier.